app/models/authentication.rb

```diff
@@ -20,6 +22,8 @@ def display_name
   protected
 
   def check_not_last
+    return if user.is_deleted?
```

Comment:
Given the above, you may want `user.nil? || user.is_deleted?` or something similar.